backend/src/main/java/feedzupzup/backend/feedback/domain/Feedback.java

```diff
@@ -93,6 +98,13 @@ public void updateLikeCount(int likeCount) {
         this.likeCount = likeCount;
     }
 
+    public void updateComment(final String value) {
+        if (this.comment == null) {
+            this.comment = new Comment();
+        }
+        this.comment.updateComment(value);
+    }
```

Comment:
p3)
```suggestion
    public void updateComment(final Comment comment) {
        this.comment = comment
    }
```
값 바꿔주는 것보다 객체 교체가 더 좋은 것 같아요!